releasetasks/templates/mobile/common_extras.yml.tmpl

```diff
@@ -14,6 +14,6 @@ build_props:
     revision: {{ revision }}
```

Comment:
I looked through the rules recently and didn't find any remotely recent changes to the fennec beta/release rules, fwiw!